CONTRIBUTING.md

```diff
@@ -52,6 +56,7 @@ ensure that your code complies with some standard style conventions.
   ```
   - Make sure that the tests pass
   ```
+  python setup.py install
   python setup.py test
```

Comment:
I is very strange, but even though I added `wget` to the `tests_requires` and pip does **download** it, Python is not able to find it when running the tests: see https://travis-ci.org/RemiLehe/openPMD-viewer/jobs/139221627

For now, I suggest just telling developers to install `wget` inside `CONTRIBUTING.md` (`pip install wget` works well). Let's try to have a PyPI version rather soon, and then we can improve it later.